toolkit/scripts/toolchain/container/toolchain_build_in_chroot.sh

```diff
@@ -1071,6 +1071,7 @@ tar xf nss-3.44.tar.gz
 pushd nss-3.44
 patch -Np1 -i ../nss-3.44-standalone-1.patch
 cd nss
+export NSS_DISABLE_GTESTS=1
 # Build with single processor due to errors seen with parallel make
 make -j1 BUILD_OPT=1                    \
```

Comment:
Do you remember why we're using only one thread here? I'm curious if this might be another opportunity to speed things up.

Reply:
According to BLFS, nss does not support parallel build.
Ref: https://www.linuxfromscratch.org/blfs/downloads/9.0-systemd/BLFS-BOOK-9.0-systemd-nochunks.html#nss